make/swagger.mk

```diff
@@ -9,9 +9,8 @@ GOSWAGGER := $(GOPATH)/bin/swagger
 $(GOSWAGGER):
 	go get -u github.com/go-swagger/go-swagger/cmd/swagger
 
-SWAGGER_CODEGEN_CLI := swagger-codegen-cli.jar
-$(SWAGGER_CODEGEN_CLI):
-	wget http://repo1.maven.org/maven2/io/swagger/swagger-codegen-cli/2.1.6/swagger-codegen-cli-2.1.6.jar -O swagger-codegen-cli.jar
+SWAGGER_CODEGEN_CLI_IMAGE := clever/swagger-codegen\:bb10ca7
+WAG_IMAGE := clever/wag\:4f8248f
```

Comment:
yeah... we could change the .drone.yml in wag to publish a "latest" tag, and then have this pull that every time... this could be good while we're iterating on it a lot. What do you think?